### ISSUE TITLE
Multiple inline configurations issue

### ISSUE THREAD

**Issue by nighthavvk** [lesshint/lesshint#452]
Hello. I face an issue while using inline configurations. It's not possible to use inline disabling more than once in one file.

**How are you running `lesshint`? CLI, Node.js API, Grunt/Gulp plugin?**
Grunt plugin: "grunt-lesshint": "^1.5.1" (https://www.npmjs.com/package/grunt-lesshint)

**What's your `.lesshintrc` configuration?**
```json
    "qualifyingElement": {
        "enabled": true,
        "allowWithAttribute": true
    },
```

**What did you expect to happen?**
```less
.tabs {
    &__list-item {
        display: none;

        // lesshint-disable-next-line qualifyingElement
        &.active { // This line won't be reported
            display: block;
        }
    }

    &__content-item {
        display: none;

        // lesshint-disable-next-line qualifyingElement
        &.active { // This line won't be reported
            display: block;
        }
    }
}
```

**What's the actual result?**
```less
.tabs {
    &__list-item {
        display: none;

        // lesshint-disable-next-line qualifyingElement
        &.active { // This line won't be reported
            display: block;
        }
    }

    &__content-item {
        display: none;

        // lesshint-disable-next-line qualifyingElement
        &.active { // This line will be reported
            display: block;
        }
    }
}
```

**Would you be interested in submitting a PR for this issue?**
Not sure I manage to do that.

**Comment by jwilsson** [comment]
Hi!
Thanks for the report and for using the template!

This is a bug, I can confirm it.

**Comment by jwilsson** [comment]
A fix for this has been released in `4.6.4`.
